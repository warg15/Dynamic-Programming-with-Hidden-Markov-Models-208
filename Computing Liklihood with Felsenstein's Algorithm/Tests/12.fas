>Brown-mesite
GAGGTAGAATGGACGCCTGGTGAGTCCTAGTTCGGTTGTTATGTCCCCTGCAAAAGTCTTCGCATGCGCCCAGCTGAAAGCTCTATGGGTAACATTAATAACTTAATAGATGTTGCATAGAAATCCAATCAAAGTTAACACATTTATTAATTGTCTAAGACTATTGACTTGACCAAGGAAGCCAAACCTCAATACATGTA
>MacQueens-bustard
GTGGTAGAATGGACGCCCGGTGAGTTCTCTTTAGGTTATTATGTGCCCTGCAATATTCTGCGCATGTACCCAGATGAGAGCTCTACGGGTAACATCGACAACCTAGCAGACGCCCCATAGGAGTCCGATAAAAGTGAGCACATTTATTAATTGTCTAAGACTATTGCCTTGAACAAGTGGGCGAAACCTCAATACATGTA
>Northern-fulmar
GTGGTAGAATGGACGCCCGGTGAGTTCTCGTTCGGTTATTAAGTCCCCTGCAAAAGTCTGCGCACGCACCCAGATGAGAGCTATATGGGTAACATCAACAACCTAACAGATGATGCATAGAAGTCCGATCAAAGTGAGCACATTTATTAATTGTCTAAGACTATTGGCTTGAGCAAGGAGGCCAAACCTCAATACATGTA
>Little-egret
ATGGTAGAATGGACGCCCGGTGAGTTCTCGTTCGGTTATTATGTCCCCTGCAACAGTCTGCGGATGCACCCAGATGAGAGCTCTATGGGTAAGATCAACAACCTAACAGATATTGCATAGAAGTACGATCAAAGTGAGCACATTTATTAATTGTCTAAGACTATTGGCTTGGGCAAGGAGGCCAAACTTCAATACATGTA
>Emperor-penguin
GTGGTAGAATGGACGTCCGGTGAGTTCTCGTTCGGTTATTATGTCCCCTGCAAAAGTCTGCGCATGCACCCAGATGAGAGCTCTATGGGTAACATCAACAACCTAACAGATGTTGCGTAGAAGTCCGATCAAAGTGAGCATATTTATTAATTGTCTAAGACTATCGGCTTGAGCAAGGAGGCCAAACCTCAATACATGTA
>Pekin-duck
GTGGTACAATGGGCAGCCGGTCAGTTCCCGCTCGGTTATTACGTCCCCAGCAAGAGTATGCGCATGCACGTAGATGAGAGCTCTATAGGCAACATCAGCAACATAATGGATGTTGCATAGAAGTCCGATCTAAGTGAGCACATTCACTAATTGTCCAAGACAATGGGCCTGAGTAAAGAAGCCGGGCCTCAATACGCGTA
>Hoatzin
GTGGTAGAATGGACGCCCGGTGAGGTCTCGTTCGGTTATTATGTCCCCTGCAAAAGTCTGCGCATGCAGCCAGATGAGAGCTCCATGGGTAACATCAACAACCTAACAGATATTGTATAGAAGTCCGATCAAAGTAAGCGCATTTATTAATTGTCTAAGACTATTTGCTTGAGCAAGGAAGCCAAACCTCAATACTTGTA
>Rhinoceros-hornbill
ATGGTAGAATGAGCGACGGGTGAGTTCTCGTTCGGTTATTATGTCCCCTGCGAAAGTCTGCGCATACACCCAGATGAAAGCGCTATGGGTAACATAAAGAACCTAACAGATGTTCCACAGAGGTCCAACGAAAGTGAGCACATTTATTAATTGTCTGAGACTATTGACTTGAGCAAGGAGGCCAAGCCTCGACACATGTC
>Yellow-throated-sandgrouse
GTGTTAGAATGGATGCACGCTGAATTCTCGTTCGGTGGTTATGTCCCCCGCAAAAGTCTACGCATGAACCCGGATGAGAGCTCTATGGGTAACATCAACAACCCAACAGATGTTGCGTAGAAGTCCGATCAAAGTGAGCACATTTATTAATCATCTAAGGCTATTGGCTTGAGCAAGGAAGCCAAGCCTCAAGACATGTA
>Adelie-penguin
GTGGTAGAATGGACGCCCGGTGAGTTCTCGTTCGGTTATTATGTCCCCTGCAAAAGTCTGCGCATGCACCCAGATGAGAGCTCTATGGGTAACATCAACAACCTAACAGATGTTGCGTAGAAGTCCGATCAAAGTGAGCATATTTATTAATTGTCTAAGACTATTGGCTTGAGCAAGGAGGCCAAACCTCAATACATGTA
>Barn-owl
GTGGTAGAATAGACGCCCGGTGAGTTCTCGTTCGGTTATTATATCTCCTGCAAAAGTCTGCACAAACACCCAGATGAGAGCTCGATGGGCAACATCAACAACCTAACAGATGTTTCATAGAAGTCCGATCAAAGTGAGCACATTTATTTGTTGTATAAGACTATTGGTTTGAGCAAGGAGGCCAAACCTCAATACATGTG
>Zebra-finch
GTGTTAGAGCGGACGCCCGGTGAGTTCTCGTTCGGTCGTTATGTATCCTGCGAAAGCTTGCGCATACACTCAGATGCGAGCTCTATGGGTAACATCAATAACCTACCCGGTGTTTCCTAGAGGTCCGATCAAAGTGAGTACATCTATTAATTATCTAACACTATTGTCTTGAGTAAGGAGACTAAACCTTAATCCATGTT
>Red-throated-loon
GTGGTAGAATGGACGCCCGGTGAGCTTTCGTTCGGTTATTATGTGCCCTGCAAAAGTCTGCGCACGCACCCAGATGAGAGCTCTATGGGTAACATCATCAACCTAACAGATGTTGCAGAGAAGTCCGATCAAAGTGAGCACATTTATTAATTGTCTAAGACTATTGGCTTGAGCACGGACGCCAAACCTCAATAGATGTA
>Carmine-bee-eater
GTGGTAAAAATGACACCCGGTGAGTTCTCATTCGGTTAATATGTCCCCTGCAAAAGCCTGCGCATACGCCCAGATGAGAGCTGTATGGGTGACATCAACAACCCGACAGATGCTTCATAGAAGTCCGGGTAGAGTGAGCACATTTATTAATTGTCTAAGGCTACTGGCTTGAGTAAGGAGGCCAAACTTCAATACATGTC
>Annas-hummingbird
GAGGTAGAATGGGCGCCCGGTGAGTTCTTGTCCGGTAACTATGTACCCTGTAAAAGTCTGCTCATTCACTCAAATCAGAGCTCTTTGGGTAACACCAACAACCTAACAGATGCTGCATAGAAGTCCGATCAAAGTGAACACATTTGTGAACTTTCTGAGACTATTCGCTTGAGCTAGGAGTTCAGATCACAATACTGATA
>Great-crested-grebe
GTGAAAAAATGGACGCCCAGTGAATTCTCGTTCGATTGTTATGTCACCTGCAAAAGTCTGCGCATGCACCCAGATGAGAGCTCTATGGGTAACATCAACAACCTAATAGATGTTGCATAGAAATCCGATCAAAGTGAGCACATTTATTAATTGTCTAAGACTACTGGCTTGAGCAAGGAGGCCAGACCTCAATACATGTA
>Cuckoo-roller
GTGGTAGAATGGACGCCCGGTGAGTTCTCGCTCGATTATTATGTCCCCTGCAAAAGTCTGCGCATACACCTAGGTGAGAGCACTATTGGAAACATCAAGAACCTAACAGATGTTTCATAGAAGTCCGATCAAGGTCAGCACATTTATTAATTGTCTAAGACCATTGGCTTGAGCAAGAAGGCCAAACCTCAATACATGTC
>Peregrine-falcon
GGGGTAGAATGGACGCCCGGTGAGTTCTCGTTCGGTTATTATGTCCCCTGCAAAAGTCTGCGCATACACCCGGATGAGAGCTGTATGGGTAACATCAACAACCTAATAGATGTTTCATAGAAGCCCGGTCAAAGTGAGCACATTTATTTATTGTCTAAGACTATTGGCTCGAGCAAGGAGGCCAAACCTTAATCCATATC
>Downy-woodpecker
GTGGTAGGACGGATGCCCGGTGAGTTCCCGGTCGGTTAATACGTCCCCCGCAAAAATCTACGCATGCACCAGGATGAGAGCTCTACGGGTAACCTCAACAACCGAACAGATGCTTCATAGAAGTCTTATCGAAGTGAGCACATCTATCAATTTTCTAGGACTATTGGCTTGAGCAAGGAGACCAAATATCAATACATGTC
>Common-cuckoo
CTGGTAGAATGGACGCCCGGAGAGTTCTCTTTCGGTTATTATGCCTCCTGCGAAAGTCTGCGCAAGCACCTAGATGACCGCCCTATGGGTAACATCAGCAGCCTAACAGATGTAGCATAGAAGTTCAATCAAAGTGAGCACATTTAGTAATTGTCTAAGGCTACTGACTTGAGCGAGGAGGCCAAACCTCAATACGTGCG
>White-throated-tinamou
ATTAGAGAACGAAGGCCAGGTGCACTCTCGTTTGGTTCTTATGTCCCCTGCAAACGTATGCAAATGCACCTCGATGAGAGTTCCATGGGTAACATCAAAAACCTGATAGATGTCGCGTAGAAGTCCGATCAAACTGAGCATACTTATTAATTACTTAAGACCATGGACTGGAGCAAGGAAGCCAAACCTCAATACATGTG
>Golden-collared-manakin
GTGGTAGAATGGACGCCCTGTGAGCCCTCGTTCGGTTGTTATATGTCCTGCAAAAGTCTGCGCATATACCCAGATGCGAGCTCTATGGATGACATCAACAACCTAGCCGATGTTTCCTAGAAGTTCGATCAAAGTGAGTACATCTATTAATTGTCTAAGACTATTGTCTTGAGTAAGGAGGCTAGACCTCAATTCATGTC
>Common-ostrich
GTGATACGATGGACGCCAGGTAAGCTCTCGTTCGGTTCCTATGTCCCTTGCAAACGTCTGCACATGCACCTAGATGAGAGCTCTATGGGTAAAATCAACAACCTAATAGATGTTGCATAGAAGTCCGATCATACTGAGCACATTTATTAATTACCTAAGGCTATGGGCTGGAGCGAGGAAGCCAAACTTCAATACATGTG
>Chuck-wills-widow
GTGGTAGAATGGGCGCCCGGTGAGTTCTCGTTCGGTCATTATATCCCCTGCGAAAGTCTGCGCATGCGCCCAGATGAGAGCTCTACGGGTAACATCAAAGACCGAACCGATGTTGCATAGAAGTCCGATCAAAGTGAACACATTTAATAATTTTCTAGGACTATTGCCTTGAGCAAGGAGGTCATACCTCAATACATCTA
>Budgerigar
GTGGTAGAATAGTCTCCCGATGAGTTCTCGTTCAGTTATTATGTCCCCTGCAAAACTCTCCGAATGCACCCAGATGAGAGCTCTATGGGTAACATCAACAACCTAGCAAATGTTTCATGGAAATCCGACCAAAATGAGCACATTTACTAATTGTGAAGGACTATTGGCTTGAGCAAGGAGGCCAAACCTCAACTAATATC
>Bar-tailed-trogon
GTGCTAGAATGGATGCCCGGTGAGTTCTCGTTCGGTGATTATGTCCCCTGCCAAAGTCTGCACATACACACAGAGCAGGGCTCTGTGGGCAACATCAACAACCTAGCAGATGTTTCATAGAAGTCCGATCAATGTGAGCACATGTATTAATTGTCTAAGACTATTGGCTTGAGCAAGGAGGGCAAACGCCAATACATGTC
>White-tailed-tropicbird
GTGGTAGAATGGACACCCGGTGAGTTCTCGTTTGGTTATTATGTCCCCTGCAAAAGTCTGCGCATGCACCCAGATGAGAGCTCTATGGGTAACATCAACAACGTAGCAGATGTTGCATAGAAGTCCGATCAAAGTGGGCACATTTATTAATTGTCTAAGACTGTTGGCTCGGGCAACGGGGCCAAACCTCAATACTTGTA
>Medium-ground-finch
GTGTTAGAGCGGACGCCCGGTGAGTTCTCGTTCGGTTGTTATGTGTCCTGCGAAAGCTTGCGCATACACTCAGATGCGAGCTCTATGGGTAACATCAATAACCTACCCGGTGTTTCCTAAAAGTCCGATCAAAGTGAGTACTTCTATTAATTGTCTAACATTATTGTCTCGAGTAAGGAGGCTAAACCTTAATCCATGTT
>White-tailed-eagle
GTGCTAGAATGGACGCCCGGTGAGTTCTCGTTCGGTTATTATGTCACCTGCAAAAGTCTGCTCATACACCCAGATGAAAGCTCTATGAGTAACATCAACAACCTAACAGGTGTTTCATAGAAGTCCGATCAAAGTAAGCACATTTATTAGTCGTATAAGATTATTGGCTTGAGCAAGGAGTCCAAACCTCAATACATTTC
>Bald-eagle
GTGCTAGAATGGACGCCCGGTGAGTTCTCGTTCGGTTATTATGTCACCTGCAAAAGTCTGCTCATACACCCAGATGAAAGCTCTATGAGTAACATCAACAACCTAACAGGTGTTTCATAGAAGTCCGATCAAAGTAAGCACATTTATTAGTCGTATAAGATTATTGGCTTGAGCAAGGAGTCCAAACCTCAATACATTTC
>American-flamingo
GTAATAGAATGGACGCCCGGTGAGTTCTCGTTCGATTGTTATGTCCCCTGCAAAAGTCTGCGCATGCATCCAGATGAGAGCTCTATGGGTAACATCAACAACCTAACAGATGTTGCATAGAAGTCCGATCAAAGTGAGCATATTTATTAGTTGGCTAAGACTACTGGCTTGAGCAAGGAGGCCAGACCTCAATACATGTA
>Kea
GTGGTAGAATAGCTGCCCGATGAGTTCTCGTTCGGTTATTATGGACCCTGCAAAAGTCTGCGAATACACCCACATGAGAGCTCTATGGGTAACATCAGTAACCTAACAGATGTTTCATAGAAGTCCGATCAAAATGAGCATATTTATTAATTGTCAGGGACTATTGGCTTGAGCAAGGAGGCCAAACCTCAACCCATGTC
>Turkey
GTGGTAGAATGGACACCCAGTCAGTTTTCGCTCAGTTATTACGTCCCTAGCAAAAGTCTGCGCATGAGCATAGATGAGAACTCTATAGGTAACGCCGGCAACCTAATAGATGTTACACAGAGGTCCGATCTAAGTGAGCACATTCATTAATTGTCTAAGACTATGAGCCCGAGTAAGGAAGCCTAACCTCAATACACTTA
>Turkey-vulture
GTGGTAGAATGGACGCCCGGTGAGTTCTCGTTCGGTTATTATGTCCCCTGCAAAAGTCTGCGCATACACCCAGATGAGAGCTCTATGGGCAACATCAAAAACCTAACAAATGTTTCATAGAAGTCCGATCAAAGTGAGCACATTTATTAATTGTCCAAGACTATTGGCTTGAGCAAGGAGGCCAAACTTCAATACATGTC
>Crested-ibis
GTGGTAGAATGGACGCCCGGTGAGTTCTTGCCCGGTTATCATGTCCCCTGCAAAGGTCTGCGCATGCACCCAGATGAGAGCTCTATAGGTAACATCAACAACCTAATAGATGTTGCATAGAAGTCCGATCATGGTAAGCACATTTATTAATTGTCTAAGACTATTGGCTTGAGCAAGGAGACAAGACCTCAATACATGTA
>Red-crested-turaco
GTGGTAGAATGGACGCCCGGTGAGTTCTCAATCGGTTATTATGTCCCCTGTAACAGTTTGCGCACGCACCCAGATGAGAGCTCTATGGGTAACGTCAACAATCTAACGGATGTTGCATAGGAGTCCAATCAAAGTGAGCACATCTATTAATTGTCTAAGACTATTGGCTTGAGCAAGGAGGCCAAAGCTCAATACATGTA
>Red-legged-seriema
GTGGTAGAATAGACGCCCGGTGAGTTCTCGTTCGGTTATTATGTTCCCTGCAAAAGTCTGCGCATACACCCAGATGAGAGCTCTATGGGTAACATCAACGACCTAACATATGTTTCCTAGAAGTCCGATCAAAGTGTGCAAATTTATTAATTGTCTAAGACTATTGGTTTGAGTGAGAAGGCCAAACCTCACTACATGTC
>Speckled-mousebird
GTGGTAGAATGGATGCCCGTTGCCTCCTGGTTCGGTCATTATATCCCCTGCGAAAGTCTGTGCATACACCTAGATGAGAGCTCTATTGATAACCTCAACCCCCTAACAGATGTTTCATAGGAGTCTGATCAAGGTGAGCACAATTACTAGTTATCTATGACTATTGGTTTGAGCAAGGAGGCCAAACCTGAATACATGTT
>Killdeer
GTGATGGAATGGACGCCCGGTGAGTTCTCGTTCGGTTATTCTGTCCCCTGCAAAAGTCTGCGCATGCACCCAGATGACAGCTCTAGGGGTGACATGAACAACCTAACAGATGTCGCATAGAAGTCCGATCAAAGTGAGCACACTAATTAATTGTCTAAGGCTATTAGGTTGAGCAAGGAGGCTAAACTTCAATACATGTA
>Chimney-swift
GCGGTACAATGGGCGCCCAGTGAGTTCTCCTTTGGTTATTATGTCCCCTGCAAAAGTCTGCTCATGCACCCACAGGAGAGCTCCATGGGTAACGTCAACAACCTCACGGATTCTTCATAGAAGTCCGATCAAAGTGAACACATTCGTGAATTGTCCAAGACTATGGGCCTGATCTAGGAGGTCAAACCAAAATACTTGTA
>Dalmatian-pelican
GTGGTAGAATGGACGCCCGGTGAGTTCTCGTTCGGTTATTATGTCCCCTGCAAGAGTTTGCGCATGCACCTAGATGAGAGCTCTATGGGTAACATCAACAACCTAACAGATGTTGCATAGAAGTCCGATCAAAGTGAGCACATTTATTGATTGTATAAGACTATTGGCTTGAGCAAGGAGGCCAAACCTCAATATATGTA
>Sunbittern
GTCGTAGAATGGACGCCCGGTGAGTACTAGCTCGGTTATTATGTCCCCTGCAAAAGTCTGCACATGCACCCAGACGAAAGCTCTATGGGTAACATTAACAACCTAACGGATGCTCCATAGAAGTCCGATCAAAGTGAGCAAATTTATTAAGTGTCTAAAACTATTGGTTTGAGCAAGGAGGCCAAACCTCAATACAGGCA
>Grey-crowned-crane
GTGGTAGAATGGACGCCCGGTGAGTTCCCGTTCGGTTCTTATGCTCCCTGCAAAAGTCTGCACATGAACTCAGATGAGAGGTCTATGGGTAACGTCAACAACCTAACAGATGTTGCATAGAGGCCCGACCAAAGTGAGCACATTTATTAATTGTCTAAAACTATTAGCTTGAGCAAGGAAGCCAAACCTCAATACATGTA
>Chicken
GTGGTAGAATGGACACCCGGTCCGTTCTTGCTCAGTTATTACGTCCCAAGCAAAAGTCTGCCCATGAACATAGATGAGAGCTCTATAGGTAACGCCGGCAACCTAAGAGATGTTACACAGAAGTCCGATCTAAGTGGGCACATTCATTAATTGTCCAAGACTATGAGCCCGAGTAAGGAAGGCTAACTTCAATACACTTA
>Great-cormorant
GTGGTGGAATTGACGCCCGGTAAGTTCTCGTTCGGTTGTTATGTCCGCTGTAAAAGTCTGCGCAGACACCCAGATGGGTGCTCTATGCGTAATATCAACAACCTAATAGATGTTGCTTAGAGGTCCGATCAAAGTGAGCACATTTATTAATTGTCTAAGACTATTGCCTTGAGCATGGAGGCCAAACCTCAATACATGTA
>Pigeon
GGGGTAGAATGGACGCCCGGTGAGTTTTCGTTCAGTTGTTGTGTTCCCTGCAAAAGTCTGTGCATGCACCCAGATGAAAGCTCTATGGGCGACATTAACAACGTACCAGATATTGCATAGAAGGCCGATCAAAGTAAGCACATTTACTAACTGCCTAAGACTATTGGCTTGGGCAAGGGAGCCAAACCTCAACATATGTA
>American-crow
GTGTTAGAATGGACGCCCGGTGAGTTCTCGTTCGGTTGTTATGTTTCCTGTAAAAGCTTGCGCATACACTCAGATGCGAGCTCTATAGGTAACATCAATAACCTACCCGGTGTTTCCTAGAAGTCCGATCAAAGTGAGCACATCTATTAATTCTCTAAGACTATTGTCTTGAGTCAGGAGACTAAACCTCAATGCATGTT
>Rifleman
GTGGTAGAATAGACGCCCGGCGAGTTCTCGTTCTGTTGTTATGTCTCTTGCAAAAGTCTGCGGATACACCCAGATGCGAGCTCTATGGGTAACATCAACAACCTAGCAGTTATCTACTAGGAGTCCGATCAAAGTGAGTGCGTTTATTAATTGTCTAAGACTATTGTCTCGAGTAAAGAGGCTAAACCTCAATACATGTC
