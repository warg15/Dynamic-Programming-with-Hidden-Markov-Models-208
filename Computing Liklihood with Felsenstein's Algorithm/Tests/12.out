-3523.5015312049964
